requirements/lint.txt

```diff
@@ -170,7 +171,6 @@ tomli==2.0.1 \
     --hash=sha256:de526c12914f0c550d15924c62d72abc48d6fe7364aa87328337a31007fe8a4f
     # via
     #   black
-    #   doc8
```

Comment:
Weird that this went away?

Reply:
Oops, got this backwards.

Reply:
tomli is a new dependency for doc8 as of 1.0.0